app/views/claims/pages/accessibility.en.html.erb

```diff
@@ -1,4 +1,5 @@
-<% content_for :page_title, "Accessibility statement for Claim funding for mentor training" %>
+<% content_for :breadcrumbs, govuk_breadcrumbs(breadcrumbs: @breadcrumbs, collapse_on_mobile: true) %>
```

Comment:
I wonder if the "partial" approach, similar to our `_primary_navigation` files might clean this up a tiny bit?

```suggestion
<%= render "breadcrumbs", breadcrumbs: @breadcrumbs %>
```

Reply:
I think it's not necessary, the abstraction that makes it clean is the govuk_breadcrumbs component I think. I think it's just adding indirection